Yoast/wpseo-woocommerce
# Throw a warning when WooCommerce is not active

## Summary

This PR can be summarized in the following changelog entry:

* Throw a warning when WooCommerce is not active.
* Fix the wording for when Yoast SEO is not active.

## Relevant technical choices:

* Moved the `Yoast_WooCommerce_SEO` class from the main plugin file to its own file.
* Moved the checks and warnings inside the `Yoast_WooCommerce_SEO` class to their own `Yoast_WooCommerce_Dependencies` class.

## Test instructions

This PR can be tested by following these steps:

* Activate the plugin, make sure WooCommerce is _not_ active, see the warning.

Fixes #354 
Fixes https://github.com/Yoast/bugreports/issues/553
